### PR TITLE
MAINT: Fix typo "Porland" -> "Portland" in `building` doc.

### PR DESCRIPTION
**Pull request by dhermes**
I'm actually not clear if it was meant to be "Borland" or "Portland Group", so this PR is not really merge-able until that can be clarified.

### CODE REVIEW

File: doc/source/user/building.rst [hunk 1]
```diff
@@ -137,7 +137,7 @@ Additional compiler flags can be supplied by setting the ``OPT``,
 Building with ATLAS support
 ---------------------------
 
-Ubuntu 
+Ubuntu
```
Comment (dhermes) [inline doc/source/user/building.rst]: This change is orthogonal to the PR. However, my editor auto-saves by stripping trailing whitespace and I figured it was worth staging the change for general hygiene. I'm happy to revert this line though if desired.

Comment (charris) [reply doc/source/user/building.rst]: No problem.

**Comment by charris** [comment]
Thanks @dhermes .

**Comment by dhermes** [comment]
Sure thing @charris. Thanks for another review!

(Not sure if you saw my comment, but I was unsure if the original intent was the "Portland Group" compiler or the "Borland" compiler.)
